docs/book/src/cronjob-tutorial/testdata/project/Makefile

```diff
@@ -82,12 +82,16 @@ test-e2e: manifests generate fmt vet ## Run the e2e tests. Expected an isolated
 	}
 	go test ./test/e2e/ -v -ginkgo.v
 
+.PHONY: lint-config
+lint-config: golangci-lint ## Verify golangci-lint linter configuration
+	$(GOLANGCI_LINT) config verify
+
```

Comment:
We change things for end users, so as it is an addition, we should use ✨ since it should be highlighted in the release notes.